### PR TITLE
Add examples and swagger folders in luis-apis published package

### PR DESCRIPTION
**Pull request by enzocanoo**
## Proposed Changes
Add `examples` and `swagger` folders in the published package based on [the feedback](https://github.com/Microsoft/botbuilder-tools/pull/433#pullrequestreview-153439394) in #433

### CODE REVIEW

File: packages/LUIS/.npmignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.luisrc
+*.tgz
+.vscode/
```
[the remark this answers is not in the document]
Comment (enzocanoo) [reply packages/LUIS/.npmignore]: The `.npmignore` is used to pack the published package, the user does not need the `.vscode` in a dependency package.

[the remark this answers is not in the document]
Comment (tomlm) [reply packages/LUIS/.npmignore]: not if we have explicit list of folders in the package.config, which we do.

File: packages/LUIS/package.json [hunk 2]
```diff
@@ -8,16 +8,17 @@
...
+    "doc": "docs",
+    "test": "test",
+    "example": "examples",
+    "bin": "bin"
   },
   "files": [
     "bin/",
     "docs/",
     "lib/",
+    "swagger/",
```
Comment (tomlm) [inline packages/LUIS/package.json]: > "swagger/", [](start = 3, length = 12)

actually, this doesn't need to be in the package.  You had it .npmignored which wasn't right. It should be in the repo but not the package.

**Comment by JuanAr** [comment]
Closing obsolete PR. This package is being published successfully.

**Pull request closed without merging**